### DISCUSSION TITLE
Welcome to webcalendar Discussions!

### DISCUSSION THREAD

**Discussion by craigk5n** [craigk5n/webcalendar#246] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.  Other users may have an answer for you.
  * Share ideas, screenshots, customization tips, etc.
  * Engage with other community members.

**Comment by lindemann06** [comment]
Is there some way to get support? No one seems to be answering questions.

**Comment by craigk5n** [comment]
Create an issue here with the "Issues" tab above.  Since this is a volunteer project, you may have to wait a bit to get a response.
